cli/src/pcluster/aws/aws_resources.py

```diff
@@ -288,22 +288,28 @@ def ec2memory_size_in_mib(self):
         return self.instance_type_data.get("MemoryInfo", {}).get("SizeInMiB")
 
 
-class FsxFileSystemInfo:
-    """Data object wrapping the result of a describe_file_systems call."""
+class FsxStorageInfo:
+    """Data object wrapping the result of a describe_file_systems and describe_file_caches call."""
 
-    def __init__(self, file_system_data):
-        self.file_system_data = file_system_data
+    def __init__(self, file_storage_data):
+        self.file_storage_data = file_storage_data
 
     @property
-    def file_system_type(self):
+    def file_storage_type(self):
```

Comment:
`FileCacheType` always returns `LUSTRE`, which seems counterintuitive since you're using FileCache and not FsxLustre. Nevertheless, this is correct because FileCache is a cache on top of a Lustre file system.
So:
1. if your logic is expecting this property to return a value representing FileCache, then we must fix it
2. It's correct for this property to be named `file_system_type` because Lustre is actually the file system under FileCache, so the property does not need to be generalized.